portal.go

```diff
@@ -508,8 +508,10 @@ func getMessageType(waMsg *waProto.Message) string {
 	switch {
 	case waMsg == nil:
 		return "ignore"
-	case waMsg.Conversation != nil, waMsg.ExtendedTextMessage != nil:
+	case waMsg.Conversation != nil:
 		return "text"
+	case waMsg.ExtendedTextMessage != nil:
+		return "post_reply"
```

Comment:
`ExtendedTextMessage` is just a text message with additional metadata, not a status reply specifically. Any text message that includes a reply, a link preview, mentions, or is a disappearing message, will be an extended text message